etas/inversion.py

```diff
@@ -877,14 +877,19 @@ def load_calculation(cls, metadata: dict):
         obj.preparation_done = True
         obj.inversion_done = True
 
-        obj.catalog = pd.read_csv(
-            obj.fn_catalog,
-            index_col=0,
-            parse_dates=["time"],
-            dtype={"url": str, "alert": str},
-        )
-        obj.catalog["time"] = pd.to_datetime(
-            obj.catalog["time"], format="ISO8601")
+        if obj.fn_catalog:
+            obj.catalog = pd.read_csv(
+                obj.fn_catalog,
+                index_col=0,
+                parse_dates=["time"],
+                dtype={"url": str, "alert": str},
+            )
+            obj.catalog["time"] = pd.to_datetime(
+                obj.catalog["time"], format="ISO8601")
+        else:
+            obj.catalog = None
+            obj.logger.warning("Catalog could not be loaded. \
```

Comment:
It would be cool if this warning could be disabled (and gets disabled when run through the entrypoints). Since I guess it is okay in the OEF case that this could not be loaded.
Otherwise, every OEF run has this warning saved with it, and somebody who doesn't know the system won't know if everything is okay...